tutorials.md

```diff
@@ -19,11 +19,15 @@ require. Don't worry though - none of them are very hard to get through. Even
 if you don't know how to use a command line or how to write Python scripts, 
 this could be a good point to start!
 
+We are also deploying the MSCP's tools in **[Materials Galaxy](https://materialsgalaxy.stfc.ac.uk/)**, 
+and there are some tutorials for using the MSCP's associated Galaxy tools.
+
 #### Basics
 
 * [Python setup](tutorial-folder/python-setup)
 * [Installing `pymuon-suite`](tutorial-folder/installing-pymuon-suite)
 * [Installing `muspinsim`](muspinsim/installation/)
+* [Installing `MuDirac`](https://muon-spectroscopy-computational-project.github.io/mudirac/installation.html)
```

Comment:
```suggestion
* [Installing `MuDirac`](mudirac/installation.html)
```
We should be able to leave out the hostname as the mudirac docs are also hosted on github.io - a relative path should work, like we use for muspinsim.